workspaces/copilot/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@internal/copilot",
+  "version": "1.0.0",
+  "private": true,
+  "engines": {
+    "node": "18 || 20"
+  },
+  "scripts": {
+    "dev": "concurrently \"yarn start\" \"yarn start-backend\"",
+    "start": "yarn workspace app start",
+    "start-backend": "yarn workspace backend start",
+    "tsc": "tsc",
+    "tsc:full": "tsc --skipLibCheck false --incremental false",
+    "build:all": "backstage-cli repo build --all",
+    "clean": "backstage-cli repo clean",
+    "test": "backstage-cli repo test",
+    "test:all": "backstage-cli repo test --coverage",
+    "fix": "backstage-cli repo fix",
+    "lint": "backstage-cli repo lint --since origin/main",
+    "lint:all": "backstage-cli repo lint",
+    "prettier:check": "prettier --check .",
+    "prettier:write": "prettier --write .",
+    "new": "backstage-cli new --scope @backstage-community",
+    "build:api-reports": "yarn build:api-reports:only --tsc",
+    "build:api-reports:only": "backstage-repo-tools api-reports -o ae-wrong-input-file-type --validate-release-tags"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "plugins/*"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/backstage/community-plugins",
+    "directory": "workspaces/copilot"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.26.11",
+    "@backstage/e2e-test-utils": "^0.1.1",
+    "@backstage/errors": "^1.2.4",
+    "@backstage/repo-tools": "^0.9.4",
+    "@changesets/cli": "^2.27.1",
+    "@spotify/prettier-config": "^15.0.0",
+    "concurrently": "^8.0.0",
+    "node-gyp": "^10.0.0",
+    "prettier": "^2.3.2",
+    "typescript": "~5.3.0"
+  },
+  "resolutions": {
+    "knex": "^3.0.0",
+    "@types/react": "^18",
+    "@types/react-dom": "^18"
```

Comment:
Can you please add a resolution for `"@microsoft/api-extractor": "7.36.4"`, this will make sure the API Reports have the correct naming convention. You'll need to delete the ones you added, sorry about that.

Reply:
Do I need run the command again?

`yarn build:api-reports`

Reply:
Yes, that will generate the reports in the correct format and then you can delete the other ones with the bad naming, please.